mne/epochs.py

```diff
@@ -2132,9 +2132,18 @@ def _read_one_epoch_file(f, tree, fname, preload):
         if len(processed) == 0:
             raise ValueError('Could not find processed data')
 
-        epochs_node = dir_tree_find(tree, FIFF.FIFFB_EPOCHS)
+        if fix_tags:
+            epochs_node = dir_tree_find(tree, 122)
+        else:
+            epochs_node = dir_tree_find(tree, FIFF.FIFFB_EPOCHS)
```

Comment:
@jaeilepp should we prefix this with `MNE_` like `MNE_FIFFB_EPOCHS`? I can't remember what would be most consistent with how the constants are currently structured. Now is the time to make as many of these things consistent as possible.